NosTests/Model/EventTests.swift

```diff
@@ -39,7 +43,21 @@ final class EventTests: CoreDataTestCase {
     
     func testIdentifierCalculationWithNoTags() throws {
         // Arrange
-        let event = try createTestEventWithNoTags(in: testContext)
+        let content = "Testing nos #[0]"
+        let event = try EventFixture.build(in: testContext, content: content)
+        
+        // Act
+        XCTAssertEqual(
+            try event.calculateIdentifier(),
+            "bc45c3ac53de113e1400fca956048a816ad1c2e6ecceba6b1372ca597066fa9a"
+        )
+    }
+
+    func testIdentifierCalculationWithEmptyTags() throws {
```

Comment:
I found a bug when creating the `EventFixture` where an `Event` with no tags and one with an empty array of tags would compute different identifiers. So I fixed that and added a test for it.

Reply:
I think it would be more communicative (and self documented) if the test includes XCTAssertNotEqual to an identifier generated by a event without tags.

Reply:
Ok, I started to do that but it made more sense to me to just have two XCTAssertEquals with different values next to them. Let me know what you think.